### PR TITLE
Schedules that start less than six weeks from now can't be edited

### PR DESCRIPTION
**Pull request by AndrewVos**
image

@guntrisoft: Not sure about that `<div><strong>` going on there...

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -27,16 +27,35 @@
   <tbody>
   <% @schedules.each do |schedule| %>
     <tr>
-      <td><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= "#{schedule.title}" %></td>
+      <td>
+        <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= "#{schedule.title}" %>
+        <% unless schedule.modifiable? %>
+          <div>
```
Comment (Guntrisoft) [inline app/views/users/edit.html.erb]: maybe use `<p><b>....</b></p>`

Comment (AndrewVos) [inline app/views/users/edit.html.erb]: done

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -27,16 +27,36 @@
...
   <% @schedules.each do |schedule| %>
     <tr>
-      <td><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= "#{schedule.title}" %></td>
+      <td>
+        <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= schedule.title %>
+        <% unless schedule.modifiable? %>
+          <p>
+            <b>
+            This schedule cannot be modified as it starts less than six weeks from now.
+            Create a new schedule to change this guiders bookable slots.
```
Comment (benbarnett) [inline app/views/users/edit.html.erb]: Tiny one: can we have an apostrophe for `guider’s bookable slots'?

Comment (AndrewVos) [inline app/views/users/edit.html.erb]: Yup done!

**Comment by Guntrisoft** [comment]
👍

**Comment by benbarnett** [comment]
Looks good to me!

**Pull request merged**